### PR TITLE
Custom shape screen

### LINKED ISSUE #259: [WEB] Shape configuration screen
- [x] Page must be accessible from landing page
- [x] User need to able to choose the number of rows / columns of the container and see the final size
- [x] User need to be able to remove "box" of the container in order to have the wanted shape for the container

### PR DESCRIPTION
**Pull request by CedricCORGE**
Possibility to disable somes locker space, and increase/decrease width/height of container

image
image
image
image

close #259

### CODE REVIEW

**Review by 31Nathan — APPROVED** [review]
Good for me !

File: Configurateur/front/lib/screens/container-creation/shape_screen.dart [hunk 1]
```diff
@@ -0,0 +1,463 @@
...
+                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                  children: [
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceAround,
+                      children: [
+                        const Text(
+                          'Largeur:',
+                          style: TextStyle(fontWeight: FontWeight.bold),
+                        ),
+                        Text('$width mètres'),
```
Comment (hugolopespinto) [inline Configurateur/front/lib/screens/container-creation/shape_screen.dart:431]: if 1 -> 'mètre' else 'mètres'

File: Configurateur/front/lib/screens/container-creation/shape_screen.dart [hunk 2]
```diff
@@ -0,0 +1,463 @@
...
+                      ],
+                    ),
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceAround,
+                      children: [
+                        const Text(
+                          'Hauteur:',
+                          style: TextStyle(fontWeight: FontWeight.bold),
+                        ),
+                        Text('$height mètres'),
```
Comment (hugolopespinto) [inline Configurateur/front/lib/screens/container-creation/shape_screen.dart:441]: if 1 -> 'mètre' else 'mètres'

**Review by hugolopespinto — CHANGES_REQUESTED** [review]

**Review by hugolopespinto — APPROVED** [review]
good for me

**Pull request merged**